_docs_operate/configuration.md

```diff
@@ -306,6 +309,115 @@ It is not recommended to use this Module for production scenarios.
 
   If set to `true` rapidoc persists the API Key in the local storage of the browser.
 
+#### messageBrokerPublisher <a href="{% link _docs_operate/modules.md %}#messagebrokerpublisher"><i class="fas fa-fw fa-info-circle"/></a> {#messagebrokerpublisher}
+
+**Sample Configuration:**
+
+```jsonc
+{
+  // ...
+
+  "modules": {
+    "messageBrokerPublisher": {
+      "enabled": false,
+      "brokers": []
+    }
+  }
+}
+```
+
+- **enabled** `default: false`
+
+  Enable or disable the sync Module.
+
+- **brokers** `default: []`
+
+  Here you can define multiple brokers to which the Connector should publish messages.
+
+  Each broker consists of a `type` (string) and a `config` object. The `type` specifies the type of the broker (e.g. `AMQP` or `PubSub`) and the `config` object contains the configuration for the broker.
+
+  - type `AMQP`
+
+    **example**
+
+    ```jsonc
+    {
+      "type": "AMQP",
+      "config": {
+        "url": "amqp://example.com:5672",
+        "exchange": "myExchange"
+      }
+    }
+    ```
+
+    **configuration**
+
+    - url `string, required` - the URL of the AMQP broker
+
+      > the URL must be in the [AMQP url format](https://www.rabbitmq.com/docs/uri-spec)
+
+    - exchange `string` - the name of the exchange to publish to
+    - timeout `number` - the timeout in milliseconds for the broker to respond
+
+  - type `MQTT`
+
+    **example**
+
+    ```jsonc
+    {
+      "type": "MQTT",
+      "config": {
+        "brokerUrl": "mqtt://example.com:1883"
+      }
+    }
+    ```
+
+    **configuration**
+
+    - brokerUrl `string, required` - the URL of the MQTT broker
+
+      > the URL can be on the following protocols: `mqtt`, `mqtts`, `tcp`, `tls`, `ws`, `wss`, `wxs` or `alis`
+
+  - type `PubSub`
+
+    **example**
+
+    ```jsonc
+    {
+      "type": "PubSub",
+      "config": {
+        "projectId": "myProjectId",
+        "topicName": "myTopicName",
+        "keyFile": "path/to/keyfile.json"
+      }
+    }
+    ```
+
+    **configuration**
+
+    - projectId `string, required` - the project id of the Google Cloud project
+    - topicName `string, required` - the name of the PubSub topic to publish to
+    - keyFile `string, required` - the location of the key file to authenticate with the Google Cloud project
```

Comment:
If a file path can be relative, I'd like to know how this path will be resolved. The two options here are (1) relative to the working directory from which the connector is started and (2) relative to the directory of the configuration file we're writing.

Would be helpful to be explicit about this here.

Reply:
1. this is only routed to the google pubsub library, so we don't know and don't care about relative urls
2. this is in docker and we define nowhere were our base path is, so this shouldn't matter

wtb just saying they should use an absolut path?

Reply:
> It's probably the working directory of the connector

and the user shouldn't care about this working directory.

> Sysadmins will hate you if you force them to use absolute paths

it's VERY common in the docker world that you have absolute paths. You mount it into the container using an absolute path, why should you care if you use an absolute path in the config?